community/event_management/web_template/schedule_section/schedule_section.html

```diff
@@ -0,0 +1,74 @@
+{% set schedule_docs = frappe.get_all("Schedule", fields =["speaker", "slot"], filters={
+'event': event
+},) %}
+{% set slot_data = {} %}
+{% for schedule in schedule_docs %}
+{% set speaker = frappe.get_doc("Speaker Registration", schedule.speaker) %}
+{% set slot = frappe.get_doc("Slot", schedule.slot) %}
+<div class="hide">{{ slot_data.setdefault(slot.date, []).append({
+	"slot": slot.name,
+	"start_time": slot.start_time,
+	"end_time": slot.end_time,
+	"speaker": speaker.name,
+	"picture": speaker.picture,
+	"about": speaker.about,
+	"title": speaker.title,
+	"full_name": speaker.full_name
+	}) }}</div>
+{% endfor %}
+<div class="courses-header">{{title}}</div>
+<div class="common-page-style">
+	<div class="container">
+		<div class="course-content-parent">
+			<div class="course-details-outline">
+				<div class="course-home-outline">
+					<div class="coure-outline">
+						<div>
+							{% for day in slot_data %}
+							<div class="small-title chapter-title" data-target="#apps-sites-and-bench"
+								data-toggle="collapse" aria-expanded="false">
+								<img class="chapter-icon" src="/assets/community/icons/chevron-right.svg" />
+								{{day}}
+							</div>
+							<div class="card-divider"></div>
+						</div>
+						{% endfor %}
+					</div>
+				</div>
+				<div>
+					{% for day in slot_data %}
+					<div class="course-details-outline">
+						<div class="course-home-headings">{{day}}</div>
+						<div class="coure-outline">
+							{% for slot in slot_data[day] %}
+							<div class="schedule-container">
+								<div class="schedule-inner-container">
+									<div class="schedule-slot">
+										{{ frappe.format(slot.start_time, {'fieldtype': 'Time'})}} - {{
+										frappe.format(slot.end_time, {'fieldtype': 'Time'}) }}
```

Comment:
use frappe.utils.format_time() instead

Reply:
Any difference between them ?

Reply:
frappe.utils.format_time() helps you format your time anyway you want.

Reply:
In that case, we can make a separate doctype for talk. This schedule doctype stores slots for each speaker, could be rendered via web template.